README.md

```diff
@@ -18,8 +18,8 @@ it needs to be included while running them.
     ```
     $ source setup.sh
     ```
-
-1. If you have, activate
+ 
+ 1. If you have, activate
```

Comment:
Please don't include extraneous whitespace changes. It messes up the blame and makes everything harder to work with in the future.

Reply:
I'll correct it right away.